Makefile

```diff
@@ -0,0 +1,10 @@
+.PHONY: test build build_linux
+
+test:
```

Comment:
can we also add run of go vet and go fmt

.PHONY: fmt
fmt:
	@out=$$(gofmt -d -e -s ${GOPKGS_DIRS} 2>&1) && [ -z "$${out}" ]  || (echo "$${out}";exit 1)

.PHONY: update-fmt
update-fmt:
	@gofmt -l -w ${GOPKGS_DIRS}

.PHONY: vet
vet:
	@go vet ${GOPKGS}

test: fmt vet
 go test...

Reply:
Done